### PR TITLE
Filled in cable soil params page

### LINKED ISSUE #122: CABLE soil params namelist file
Need a description of cable soil parameters namelist file.
The Jules documentation already has a good description here:
http://jules-lsm.github.io/latest/namelists/cable_soilparm.nml.html

### PR DESCRIPTION
**Pull request by tammasloughran**
# CABLE

Thank you for submitting a pull request to the CABLE Project.

## Description

This pull request adds an example namelist file for the CABLE soil parameters and a table that describes the namelist variables.
Some data has been taken from the JULES documentation http://jules-lsm.github.io/latest/namelists/cable_soilparm.nml.html

Fixes #122

## Type of change

- [ ] New or updated documentation

## Checklist

- [x] The new content is accessible and located in the appropriate section.
- [x] I have checked that links are valid and point to the intended content.
- [x] I have checked my code/text and corrected any misspellings

### CODE REVIEW

File: documentation/docs/user_guide/inputs/cable_soilparm_nml.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+| soilin%ssat       | real(n_soiltypes) | >=0.0            | uninitialised  | Volume of H2O at saturation \( (m^{3} \cdot m^{-3}) \).                                  |
+| soiln%hyds        | real(n_soiltypes) | >=0.0            | uninitialised  | Hydraulic conductivity at saturation \( (m^{-1}) \).                                     |
+| soilin%sucs       | real(n_soiltypes) | <=0.0            | uninitialised  | Suction at saturation \( (m) \).                                                         |
+| soilin%rhosoil    | real(n_soiltypes) | >=0.0            | uninitialised  | Soil bulk density \( (kg \cdot m^{-3}) \).                                               |
+| soilin%css        | real(n_soiltypes) | >=0.0            | uninitialised  | Soil specific heat capacity \( (J \cdot kg^{-1} \cdot K^{-1}) \).                        |
+
+
+1. Brooks, R. H., and Corey, A. T. 1964. Hydraulic Properties of Porous Media. Fort Collins: Colorado State University.
+2. Campbell, G. S. 1974. A simple method for determining unsaturated conductivity from moisture retention data. Soil Science, 117(6), 311-314. [doi:10.1097/00010694-197406000-00001
```
Comment (ccarouge) [inline documentation/docs/user_guide/inputs/cable_soilparm_nml.md:58]:
```suggestion
2. Campbell, G. S. 1974. A simple method for determining unsaturated conductivity from moisture retention data. Soil Science, 117(6), 311-314. (doi:10.1097/00010694-197406000-00001)[https://doi.org/10.1097/00010694-197406000-00001]
```

**Review by ccarouge — CHANGES_REQUESTED** [review]

File: documentation/docs/user_guide/inputs/cable_soilparm_nml.md [hunk 2]
```diff
@@ -1 +1,55 @@
-# cable_soilparm.nml options
\ No newline at end of file
+# cable_soilparm.nml options
+
+!!! Note
+
+    The asterisk symbol does not mean arithmetic multiplication in the namelist file.
+    It means that the following sequence is repeated multiple times.
```
[the remark this answers is not in the document]
Comment (tammasloughran) [reply documentation/docs/user_guide/inputs/cable_soilparm_nml.md]: Cannot apply requested change from outdated commit. I added this to the latest commit. Thanks!

File: documentation/docs/user_guide/inputs/cable_soilparm_nml.md [hunk 3]
```diff
@@ -1 +1,55 @@
...
+
+!!! Note
+
+    In JULES, unintialised values are assigned as a missing data indicator (MDI)
```
[the remark this answers is not in the document]
Comment (tammasloughran) [reply documentation/docs/user_guide/inputs/cable_soilparm_nml.md]: Cannot apply requested change from outdated commit. I added this to the latest commit. Thanks!

File: documentation/docs/user_guide/inputs/cable_soilparm_nml.md (hunk 1 above)
Comment (tammasloughran) [reply documentation/docs/user_guide/inputs/cable_soilparm_nml.md]: Cannot apply requested change from outdated commit. I added this to the latest commit. Thanks!

**Review by ccarouge — APPROVED** [review]

**Pull request merged**